### DISCUSSION TITLE
Web interface, Topology hangs?

### DISCUSSION THREAD

**Discussion by olavt** [openthread/openthread#9680] [General]
I have followed the instructions here for running OpenThread Border Router in a Docker container on a Raspberry Pi 4 running the Raspberry Pi OS Lite (64-bit).

After the installation I'm able to open the OpenThread Border Router web interface on http://<Raspberry Pi Ip address>:8080/

I'm also able to use Form to form a new thread network.

When looking at the Status, it looks normal to me.

However when I select "Topology" from the web interface menu, I get only a spinning wheel looping forever.

I'm also not able to commission a Matter over Thread devices to the network (by running chip-tool) on another Raspberry Pi. If I run the OpenThread Border Router locally directly on the Raspberry Pi, I'm able to commission a Matter over Thread device to the network.

How can I troubleshoot?

**Comment by Vyrastas** [comment]
Since this is not specific to the content of docs in ot-docs, but more of a help question, I've converted this to a Discussion for better visibility.

**Comment by colonelpanic8** [comment]
well, FWIW I think I"m having the same issue

**Comment by Vyrastas** [comment]
@chshu Do you know who can help confirm/troubleshoot this?

**Comment by jgpazos70** [comment]
The same to me. Full Web GUI working (Joined network successful, State correct,...) except Topology (spinning wheel looping forever). 

ot-br-posix native installation working good (no errors) on Raspberry Pi 3B+ with Makerdiary MDK USB Dongle flashed with RCP (ot-nrf52840).
